app/login/page.js

```diff
@@ -24,18 +30,26 @@ const LoginPage = () => {
           </h1>
         </div>
 
-        <p className='josefin-sans pr-3 text-[13.47px] font-medium text-[#364C6F]'>
+        <p className='pr-3 text-[13.47px] font-medium text-[#364C6F]'>
           Get set Funded!
         </p>
       </div>
 
       <div className='mt-[90px] flex w-full flex-col items-center'>
-        <button
-          className='w-full rounded-[5px] border border-[#97BFD7] p-3 text-sm font-medium text-[#364C6F] transition-all hover:shadow-sm hover:shadow-[#97bed781]'
-          onClick={handleGoogleLogin}
-        >
-          Login with Google
-        </button>
+        <div className='relative w-full'>
+          <h1
+            className={`absolute -top-8 w-full rounded-[4px] p-1 text-center text-sm font-medium capitalize text-red-700 ${showLoginError ? 'block' : 'hidden'}`}
+          >
+            Failed to log in !
+          </h1>
+
+          <button
+            className='w-full rounded-[5px] border border-[#97BFD7] p-3 text-sm font-medium text-[#364C6F] transition-all hover:shadow-sm hover:shadow-[#97bed781]'
+            onClick={handleGoogleLogin}
```

Comment:
Did you meant to use handleGoogleSignin instead of handleGoogleLogin?

Reply:
Yes, this is a mistake from my side. I was suppose to use handleGoogleSignin instead of handleGoogleLogin.